styles/index.less

```diff
@@ -25,16 +25,15 @@ a {
   align-items: center;
 }
 
-#home {
-  margin-top: 60px;
-}
-
 .headline {
-  margin-top: 50px;
-
-  color: grey;
-  font-size: 14px;
-  letter-spacing: 0.7em;
+  margin-top: 55px;
+  margin-bottom: 8px;
+  padding-bottom: 3px;
+  border-bottom: 1px solid #aaa;
+
+  color: #333;
+  font-size: 16px;
```

Comment:
I want to humbly suggest that its a chicken and egg situation.
I would finalize these values and make variables for them >> if they looked good.
For which I need to test >> for which you need to merge.

Is that totally terrible?